### PR TITLE
ebay-video: added ability to input shaka config

### PR DESCRIPTION
**Pull request by JereMIbq1995**
## Description

This PR allows the component to read from input.shakaConfig onCreate, and configure shaka with that config once shaka has successfully loaded.

## Context

Currently <ebay-video> doesn't have the functionality to allow inputting Shaka configuration.
This creates an issue where the host of the component would see a bunch of warnings in the console if it tries to play videos of DASH (HLS) type because shaka is not configured correctly for this.
This PR allows users of the component to input shaka config in order to fix this

### Warnings:
Screenshot 2024-10-28 at 6 01 11 PM
Screenshot 2024-10-28 at 6 01 39 PM

### Usage after fix this:
Screenshot 2024-10-28 at 6 02 35 PM
Screenshot 2024-10-28 at 6 02 22 PM

### Storybook update:
Screenshot 2024-10-29 at 3 55 27 PM

## References

ebay-video HLS streaming issue: https://github.com/eBay/ebayui-core/issues/2307

### CODE REVIEW

**Comment by JereMIbq1995** [comment]
Simplified PR so it's a 1 line change

File: src/components/ebay-video/video.stories.ts
Comment (LuLaValva) [inline src/components/ebay-video/video.stories.ts:94]: Can you provide a link so people know where to find the options here?
```suggestion
                "The Shaka player [configuration object](https://shaka-player-demo.appspot.com/docs/api/tutorial-config.html). This allows users to control Shaka player.",
```

**Review by LuLaValva — COMMENTED** [review]
Just one quick documentation fix, but otherwise looks great! Thanks for the contribution!

**Comment by LuLaValva** [comment]
Also, @JereMIbq1995 before you merge can you `npm run change` and select `patch`? This is good for tracking updates (and you'll get your name on the release notes 😏)

File: src/components/ebay-video/video.stories.ts
Comment (JereMIbq1995) [reply src/components/ebay-video/video.stories.ts]: Good idea! Done 👍

**Review by LuLaValva — APPROVED** [review]

**Review by agliga — COMMENTED** [review]
Overall, im fine approving and merging when the Standard story does not have `shakaConfig` value. 
We can leave it in the IOS story

File: src/components/ebay-video/video.stories.ts
Comment (JereMIbq1995) [reply src/components/ebay-video/video.stories.ts]: Removed shakaConfig value from standard story 👍

**Review by agliga — APPROVED** [review]

**Pull request merged**
